package.json

```diff
@@ -24,7 +24,7 @@
     "@mattermost/react-native-emm": "1.5.1",
     "@mattermost/react-native-network-client": "1.8.0",
     "@mattermost/react-native-paste-input": "0.8.1",
-    "@mattermost/react-native-turbo-log": "0.5.0",
+    "@mattermost/react-native-turbo-log": "github:larkox/react-native-turbo-log#b00c042b6a580e77d509e37bcbc701e0b9383e75",
```

Comment:
Alright we need to move the pod from repo and then release this one so that we can update here before merging